### ISSUE TITLE
update ros wiki

### ISSUE THREAD

**Issue by jihoonl** [turtlebot/turtlebot_create#13]
http://wiki.ros.org/turtlebot_create

**Comment by corot** [comment]
done, isn't it?
something more to add?

**Comment by bit-pirate** [comment]
No, since the pages are mostly empty. This issue is more a reminder to fill in missing information when people have time to.

**Comment by corot** [comment]
헐...  sorry,  I just checked that it match with groovy doc

2013/9/24 Marcus Liebhardt notifications@github.com
